components/src/core/InfoListItem/InfoListItem.tsx

```diff
@@ -144,22 +163,18 @@ export const InfoListItem: React.FC<InfoListItemProps> = (props) => {
 
     const getRightComponent = (): JSX.Element | undefined => {
         if (rightComponent) {
-            return (
-                <div style={{ flex: '0 0 auto', marginLeft: 16, display: 'flex', alignItems: 'center' }}>
-                    {rightComponent}
-                </div>
-            );
+            return <div className={clsx(defaultClasses.rightComponent, classes.rightComponent)}>{rightComponent}</div>;
         } else if (chevron) {
             return (
-                <ListItemSecondaryAction style={{ display: 'flex' }}>
+                <ListItemSecondaryAction className={defaultClasses.chevron}>
                     <Chevron color={'inherit'} />
                 </ListItemSecondaryAction>
             );
         }
     };
 
     const interpunct = (): JSX.Element => (
-        <Typography className={`${classes.withSmallMargins} ${classes.separator}`}>
+        <Typography className={`${defaultClasses.withSmallMargins} ${defaultClasses.separator}`}>
```

Comment:
we can use clsx here

Reply:
do we want to expose a class here?

Reply:
I've removed the withSmallMargins class, combined it with the separator class, and made a class override for it.